doc/api/events.md

```diff
@@ -873,6 +873,25 @@ async function run() {
 run();
 ```
 
+The special handling of the `'error'` event is only used when `events.once()`
+is used to wait for another event. If `events.once()` is used to wait for the
+'`error'` event itself, then it is treated as any other kind of event without
+special handling:
+
+```js
+const { EventEmitter, once } = require('events');
+
+const ee = new EventEmitter();
+
+once(ee, 'error')
+  .then((err) => console.log('ok', err.message))
+  .catch((err) => console.log('error', err.message));
+
+ee.emit('error', new Error('boom'));
+
+// Prints: ok boom
```

Comment:
This code prints "ok undefined" in Node.js `v14.5.0` and `v12.18.2`
```suggestion
// Prints: ok undefined
```

Reply:
Correction posted by @lpinca in https://github.com/nodejs/node/pull/34225/files#r451962061